src/workloads/query/TimeseriesBlockProcessing.yml

```diff
@@ -202,6 +202,39 @@ Actors:
                     { $count: "count" },
                   ]
 
+# This query stresses the bucket level filter on nested fields.
```

Comment:
Verified this locally. Around 2-3% of buckets pass the bucket level filter.

Reply:
The convention in the TsbsOptimization is to include the stats of the bucket level filter in the [actor comment](https://github.com/mongodb/genny/blob/33839588affe9276eb7cc2d3d46592023dc56989/src/workloads/query/TimeseriesTsbsOptimizations.yml#L207-L208) but none of the other queries in this workload have that so its probably fine as is.